app/assets/stylesheets/application.scss

```diff
@@ -131,6 +135,25 @@ div.navbar-collapse {
   font-size: 18px;
 }
 
+.landing-page-bg {
+  // background-size: contain;
+  // object-fit: contain;
+  // object-fit: cover;
+  width: 100vw;
+  height: 100vh;
+  z-index: -1;
+  // object-position: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+// @media screen and (max-width: 500px) {
```

Comment:
Same here...however, it does look like responsive code. Do you need this anymore?